style.css

```diff
@@ -7,11 +7,13 @@ header {
   display: flex;
   flex-direction: row;
   align-items: center;
-  justify-content: space-between;
+  justify-content: space-around;
   width: 100%;
   height: 48px;
   margin-left: 6px;
   margin-top: -7px;
+  gap: 38%;
```

Comment:
- [ ] Please fix your mobile version, you should keep it working as well , please fix it 


![image](https://user-images.githubusercontent.com/75126481/229279469-2eba4b3b-f2cb-4845-b05d-dd4794084f27.png)